backend/battle/battles/base_stats.py

```diff
@@ -33,3 +34,18 @@ def sum_all_pokemons(pokemons):
 def sumValid(pokemon):
     sum_result = pokemon["attack"] + pokemon["defense"] + pokemon["hp"]
     return sum_result
+
+
+def verify_team_exists(battle, trainer):
```

Comment:
Could you please clarify what you're doing here? The name isn't very readable to me. This doesn't simply verifies, but also returns the pokemon team as a dict.

Reply:
Sorry, I'm going to change this function name. This function get all pokemons related to a team, if the team doesn't exist, it will be returned a default value to a dict.